plugin/trino-elasticsearch/src/main/java/io/trino/plugin/elasticsearch/AwsSecurityConfig.java

```diff
@@ -1,100 +0,0 @@
-/*
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package io.trino.plugin.elasticsearch;
-
-import io.airlift.configuration.Config;
-import io.airlift.configuration.ConfigDescription;
-import io.airlift.configuration.ConfigSecuritySensitive;
-import io.airlift.configuration.DefunctConfig;
-
-import javax.validation.constraints.NotNull;
-
-import java.util.Optional;
-
-@DefunctConfig("elasticsearch.aws.use-instance-credentials")
```

Comment:
Is there a way to preserve that `@DefunctConfig`? (And also, add some more properties to it?)

Reply:
I guess I can just add them to `ElasticsearchConfig`.